src/index.ts

```diff
@@ -236,10 +237,7 @@ prog
         module.exports = require('./${safeVariableName(
           opts.name
         )}.cjs.development.js')
-      }`,
-          {
-            overwrite: true,
-          }
```

Comment:
This isn't specified in `@types/node`. Is this a feature from `fs-extra`?

Reply:
Yep

Reply:
Added it back 👍